extensions/wikia/DesignSystem/scripts/DesignSystemGlobalNavigationScrollToLink.js

```diff
@@ -0,0 +1,40 @@
+/*global require*/
+require(
+	['wikia.scrollToLink', 'wikia.window', 'jquery', 'wikia.browserDetect'],
+	function(scrollToLink, win, $, browserDetect) {
+		'use strict';
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Use of JavaScript strict mode may result in unexpected behaviour in some browsers. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=javascript%3AStrictMode)